tests/unit_test/private/fed/app/simulator/simulator_runner_test.py

```diff
@@ -113,8 +113,8 @@ def test_split_names(self, client_names, gpus, expected_split_names):
         ],
     )
     def test_split_gpus_success(self, gpus, expected_gpus):
-        splitted_gpus = split_gpus(gpus)
-        assert splitted_gpus == expected_gpus
+        split_gpus = split_gpus(gpus)
+        assert split_gpus == expected_gpus
```

Comment:
this one should not be modified to this, suggest a different name, 
notice this will make var the same name as method "split_gpus"